src/commands/deployment/README.md

```diff
@@ -27,6 +27,39 @@ datadog-ci deployment mark --env prod --service payment-service --revision v1.1.
 - `--tags` is an array of key value pairs in the format `key:value`. These tags are added to the deployment event shown in Datadog.
 - `--no-fail` (default: `false`) will prevent the deployment command from failing if there are issues submitting the data.
 
+### `correlate`
+
+The `correlate` command "connects" the CD deployments with a CI pipeline. If you are using `datadog-ci deployment mark` then you do not need to use 
+this command and the correlation will happen automatically. This command does not work for every setup, check the section 
+[when to call the command][#When-to-call-the-command] for more details.
+
+This command will allow Datadog to correlate the associated deployments to a pipeline from the UI. It also works the other way around, 
+so that given a deployment you can check the pipeline that triggered it.
```

Comment:
I think this is clearer
```suggestion
**Note**: If you are using `datadog-ci deployment mark` then you do not need to use this command and the correlation will happen automatically.

The `correlate` command connects CD deployments with the corresponding CI pipelines: you'll be able to check what pipeline triggered a deployment and what deployments were triggered by a pipeline. 

**Important**: This command does not work for every setup, check the section [when to call the command][#When-to-call-the-command] for more details.
```
this makes me wonder: if `mark` does the job already, why do we need the extra `correlate`? It's not clear from these instructions